### ISSUE TITLE
Spoof

### ISSUE THREAD

**Issue by ankarastrix** [lshbluesky/OC-GA-B250M-DS3H-Hackintosh#21]
I also have an i3 7100, but your spoofed Iris graphics doesn't work (black screen). It shows screen instalation when I start the USB for installation, but later the monitor simply turns off.It works even without spoofing because there is support for Kaby Lake and Kaby Lake-R (for now).

**Comment by lshbluesky** [comment]
On recent macOS, the Intel HD 630 iGPU has a black screen problem and requires the use of `igfxagdc=0` (or disable-agdc=01000000 property).
In addition, since the type of display cable for each user's monitor is different, a connector patch should be applied if necessary.

Although there is a `device-id` property, the value is `12590000`, so it is not a spoof and can be deleted.
I just used the Platform ID as `0x59260002`.
If `0x59260002` doesn't work, use `0x59120000`.
The model property is just a cosmetic that only modifies the model name.

In summary, each user has different detailed specifications such as the cable type of the monitor, so use it after modifying it if necessary.
